cypress/integration/login.js

```diff
@@ -11,14 +11,16 @@ describe('Login functionality', function() {
         cy.get('[id="loginForm.userId"]').click({force:true})
         cy.get('[aria-label="Demo User"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        var currentDay = new Date().getDate()
+        cy.get('[data-test="timesheets-current-day"]').contains(currentDay)
```

Comment:
1. Is this a unique locator? [data-test="timesheets-current-day"]

Reply:
it is not unique. changed to calendar--selected